ui/packages/shared/client/src/parca/query/v1alpha1/query_pb.ts

```diff
@@ -0,0 +1,2015 @@
+proto3.util.setEnumType(QueryRequest_ReportType, "parca.query.v1alpha1.QueryRequest.ReportType", [
+  { no: 0, name: "REPORT_TYPE_FLAMEGRAPH_UNSPECIFIED" },
+  { no: 1, name: "REPORT_TYPE_PPROF" },
+  { no: 2, name: "REPORT_TYPE_TOP" },
+  { no: 3, name: "REPORT_TYPE_CALLGRAPH" },
+]);
+
+/**
+ * Top is the top report type
+ *
+ * @generated from message parca.query.v1alpha1.Top
+ */
+export class Top extends Message<Top> {
+  /**
+   * list are the list of ordered elements of the table
+   *
+   * @generated from field: repeated parca.query.v1alpha1.TopNode list = 1;
+   */
+  list: TopNode[] = [];
+
+  /**
+   * reported is the number of lines reported
+   *
+   * @generated from field: int32 reported = 2;
+   */
+  reported = 0;
+
+  /**
+   * total is the number of lines that exist in the report
+   *
+   * @generated from field: int32 total = 3;
+   */
+  total = 0;
+
+  /**
+   * unit is the unit represented by top table
+   *
+   * @generated from field: string unit = 4;
+   */
+  unit = "";
+
+  constructor(data?: PartialMessage<Top>) {
+    super();
+    proto3.util.initPartial(data, this);
+  }
+
+  static readonly runtime = proto3;
+  static readonly typeName = "parca.query.v1alpha1.Top";
+  static readonly fields: FieldList = proto3.util.newFieldList(() => [
+    { no: 1, name: "list", kind: "message", T: TopNode, repeated: true },
+    { no: 2, name: "reported", kind: "scalar", T: 5 /* ScalarType.INT32 */ },
+    { no: 3, name: "total", kind: "scalar", T: 5 /* ScalarType.INT32 */ },
+    { no: 4, name: "unit", kind: "scalar", T: 9 /* ScalarType.STRING */ },
+  ]);
+
+  static fromBinary(bytes: Uint8Array, options?: Partial<BinaryReadOptions>): Top {
+    return new Top().fromBinary(bytes, options);
+  }
+
+  static fromJson(jsonValue: JsonValue, options?: Partial<JsonReadOptions>): Top {
+    return new Top().fromJson(jsonValue, options);
+  }
+
+  static fromJsonString(jsonString: string, options?: Partial<JsonReadOptions>): Top {
+    return new Top().fromJsonString(jsonString, options);
+  }
+
+  static equals(a: Top | PlainMessage<Top> | undefined, b: Top | PlainMessage<Top> | undefined): boolean {
+    return proto3.util.equals(Top, a, b);
+  }
+}
+
+/**
+ * TopNode is a node entry in a top list
+ *
+ * @generated from message parca.query.v1alpha1.TopNode
+ */
+export class TopNode extends Message<TopNode> {
+  /**
+   * meta is the metadata about the node
+   *
+   * @generated from field: parca.query.v1alpha1.TopNodeMeta meta = 1;
+   */
+  meta?: TopNodeMeta;
+
+  /**
+   * cumulative is the cumulative value of the node
+   *
+   * @generated from field: int64 cumulative = 2;
+   */
+  cumulative = protoInt64.zero;
+
+  /**
+   * flat is the flat value of the node
+   *
+   * @generated from field: int64 flat = 3;
+   */
+  flat = protoInt64.zero;
+
+  /**
+   * diff is the diff value between two profiles
+   *
+   * @generated from field: int64 diff = 4;
+   */
+  diff = protoInt64.zero;
```

Comment:
That looks nice, it automatically handles the `int64` support: https://github.com/bufbuild/protobuf-es/blob/main/packages/protobuf/src/proto-int64.ts

The config was manual with the other plugin: https://github.com/timostamm/protobuf-ts/blob/master/MANUAL.md#bigint-support

The strict mode PR (#1632) surfaces some potential issues around it, and you might need to handle them, a large comment in the plugin's code seems to explain how